_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
The HQ is organized at two levels. First, there are features, they came from what we desire for web apps to be. The second layer are technologies, these support features, they allow the features to be real.

After reading your thoughts, I agree on removing freshness. Mainly because:
> In which case, fresh doesn't seem like an accurate differentiator, because both GOWS and PWAs are fresh, even if they use different technologies to maintain freshness.

At the same time, I don't know if _lifecycle-customizable_ (or whatever the adjective for that is) is actually a feature we want from PWA. Given this, I would prefer to simply remove fresh from features.